scripts/apache/get_fasta_sequence

```diff
@@ -28,27 +29,60 @@ sub get_fasta_sequence {
     my ($server) = @_;
 
     my $id = $server->require_argument('id');
+
+    my @raw_id_array = split (',', $id);
+    $id = "";
```

Comment:
why are you doing this?
it looks highly unconventional, perhaps the variable naming is misleading?
What comes from $server->require_argument('id') is NOT the id?
is it the id but needs processing? maybe that can be done in a function? (it looks like similar processing is performed later on?)

Reply:
It is id, just sometimes it is several ids, spited by comma, later we cut version from each id with  isoforms and glue them back to one string.